extensions/wikia/Forum/ForumBoardInfo.class.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+class ForumBoardInfo {
+
+	private $id;
+	private $name;
+	private $url;
+	private $description;
+	private $postCount;
+	private $threadCount;
+	/** @var ForumPostInfo $lastPost */
+	private $lastPost;
+
+	/**
+	 * ForumBoardInfo constructor.
+	 * @param array $array ;
+	 */
+	public function __construct( $array = [] ) {
```

Comment:
Let's enforce an `array` type of constructor argument and skip the check below. Do not fail silently.